checks/pinned_dependencies_test.go

```diff
@@ -483,26 +554,26 @@ func TestDockerfileInsecureDownloadsLineNumber(t *testing.T) {
 			endLine   uint
 		}
 	}{
-		// {
-		// 	name:     "dockerfile downloads",
-		// 	filename: "./testdata/Dockerfile-download-lines",
-		// 	expected: []struct {
-		// 		snippet   string
-		// 		startLine uint
-		// 		endLine   uint
-		// 	}{
-		// 		{
-		// 			snippet:   "curl bla | bash",
-		// 			startLine: 35,
-		// 			endLine:   36,
-		// 		},
-		// 		{
-		// 			snippet:   "pip install -r requirements.txt",
-		// 			startLine: 41,
-		// 			endLine:   42,
-		// 		},
-		// 	},
-		// },
+		{
+			name:     "dockerfile downloads",
```

Comment:
these lines we commented out as a mistake in the code. I've simply uncommented them.